apps/dashboard/app/javascript/packs/application.js

```diff
@@ -18,7 +18,7 @@ import jQuery from 'jquery';
 import 'jquery-ujs';
 import datatables from 'datatables.net';
 import datatablesBs4 from 'datatables.net-bs4/js/dataTables.bootstrap4';
-import datatablesSelect from 'datatables.net-select';
+import datatablesSelect from 'datatables.net-select/js/dataTables.select';
```

Comment:
This is the update that'll fix updating. Newer dataTables compile the wrong js file without referencing the full path.